gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -793,6 +911,73 @@ public static LicenseOption fields(LicenseField... fields) {
   }
 
   /**
+   * Class for specifying operation get options.
+   */
+  class OperationOption extends Option {
+
+    private static final long serialVersionUID = -4572636917684779912L;
+
+    private OperationOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify the operation's fields to be returned by the RPC call. If this
+     * option is not provided, all operation's fields are returned. {@code OperationOption.fields}
+     * can be used to specify only the fields of interest. {@link Operation#operationId()} is
+     * always returned, even if not specified.
+     */
+    public static OperationOption fields(OperationField... fields) {
+      return new OperationOption(ComputeRpc.Option.FIELDS, OperationField.selector(fields));
+    }
+  }
+
+  /**
+   * Class for specifying operation list options.
+   */
+  class OperationListOption extends Option {
+
+    private static final long serialVersionUID = -1509532420587265823L;
+
+    private OperationListOption(ComputeRpc.Option option, Object value) {
+      super(option, value);
+    }
+
+    /**
+     * Returns an option to specify a filter on the operations being listed.
+     */
+    public static OperationListOption filter(OperationFilter filter) {
+      return new OperationListOption(ComputeRpc.Option.FILTER, filter.toPb());
+    }
+
+    /**
+     * Returns an option to specify the maximum number of operations to be returned.
+     */
+    public static OperationListOption pageSize(long pageSize) {
+      return new OperationListOption(ComputeRpc.Option.MAX_RESULTS, pageSize);
+    }
+
+    /**
+     * Returns an option to specify the page token from which to start listing operations.
+     */
+    public static OperationListOption startPageToken(String pageToken) {
+      return new OperationListOption(ComputeRpc.Option.PAGE_TOKEN, pageToken);
+    }
+
+    /**
+     * Returns an option to specify the operation's fields to be returned by the RPC call. If this
+     * option is not provided, all operation's fields are returned.
+     * {@code OperationListOption.fields} can be used to specify only the fields of interest.
+     * {@link Operation#operationId()} is always returned, even if not specified.
+     */
+    public static OperationListOption fields(OperationField... fields) {
+      StringBuilder builder = new StringBuilder();
+      builder.append("items(").append(OperationField.selector(fields)).append("),nextPageToken");
```

Comment:
cross referencings #456 to suggests that the shared selector method that would be used in this case could get the StingBuilder as the first argument and append to it.